github_stargazers/github_stargazers.py

```diff
@@ -49,7 +49,7 @@ def process(self) -> None:
                 stargazers: typing.List[str] = github.get_all_stargazers()
                 _OutputPrintable.print_stargazers(stargazers)
         except (TooManyRequestsHttpError, UrlNotFoundError,
-                MissingHyperlinkTagError, MissingHrefAttributeError) as exception_message:
+                MissingHyperlinkTagError, MissingHrefAttributeError, HrefContentError) as exception_message:
```

Comment:
Do you think is fine or I should refactor this _except_ because of the so many Exceptions?

Reply:
This is ok since you only print the message any case.